Project.toml

```
@@ -22,13 +21,12 @@ SymbolicIndexingInterface = "2efcf032-c050-4f8e-a9bb-153293bab1f5"
 [compat]
 FastBroadcast = "0.3.5"
 LinearAlgebra = "1"
-LinearSolve = "2.32"
+LinearSolve = "3.7"
```

```suggestion
LinearSolve = "3.7.1"
```

once https://github.com/SciML/LinearSolve.jl/pull/591 is merged and a new release of LinearSolve.jl is published

---

Done!